common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -40,7 +40,7 @@ class Group(namedtuple("Group", "id name")):
 
     def __new__(cls, id, name):
         # pylint: disable=super-on-old-class
-        return super(Group, cls).__new__(cls, int(id), name)
+        return super(Group, cls).__new__(cls, id, name)
```

Comment:
@andy-armstrong @jimabramson Current implementation of user partition/group prevent from making our custom format string id's e.g., `verification:{location}`, `non_verified`, `verified_allow`, `verified_deny`.
Is it okay if we remove this `int` restriction over id's?